src/_pytest/mark/__init__.py

```diff
@@ -14,7 +18,11 @@
 __all__ = ["Mark", "MarkDecorator", "MarkGenerator", "get_empty_parameterset_mark"]
 
 
-def param(*values, **kw):
```

Comment:
I expanded the kwargs so I can annotate them and so that the types show up in the docs.